app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -498,13 +503,32 @@ function AcuantCapture(
     }
   }
 
+  function onSelfieCaptureOpen() {
+    trackEvent('idv_sdk_selfie_image_capture_opened');
```

Comment:
Same here event name?

Reply:
As mentioned above, [it is the new naming standard](https://github.com/18F/identity-idp/pull/9616/commits/3d4b4d77ddad9f1dd030463e11d61fa0a518dc9f).